exercises/bracket-push/lib/example.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:collection';
+
+class BracketPush {
+  bool isPaired(String input) {
+    ListQueue<String> stack = new ListQueue(input.length);
+
+    for (int i = 0; i < input.length; i++) {
```

Comment:
Instead of having a counter to index the characters in the string. I recommend using the pattern `for (String [insert-name] in input.split('')) {`. It shows off Dart's forEach capability and the built-in String method `split`.